README.md

```diff
@@ -101,6 +101,200 @@ any zone.
 
 You can define an arbitrary number of drag items.
 
+Analytics Events
+----------------
+
+The following analytics events are provided by this block.
+
+## `xblock.drag-and-drop-v2.loaded`
+
+Fired when the Drag and Drop XBlock is finished loading.
+
+### Example
+
+```
+{
+    "username": "staff",
+    "event_type": "xblock.drag-and-drop-v2.loaded",
+    "ip": "10.0.2.2",
+    "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:43.0) Gecko/20100101 Firefox/43.0",
+    "host": "precise64",
+    "referer": "http://localhost:8000/courses/course-v1:DnD+DnD+DnD/courseware/ec546c58d2f447b7a9223c57b5de7344/756071f8de7f47c3b0ae726586ebbe16/1?activate_block_id=block-v1%3ADnD%2BDnD%2BDnD%2Btype%40vertical%2Bblock%40d2fc47476ca14c55816c4a1264a27280",
```

Comment:
@Kelketek Is this an Apros URL? In the LMS I only get 

    http://localhost:8000/courses/course-v1:DnD+DnD+DnD/courseware/ec546c58d2f447b7a9223c57b5de7344/756071f8de7f47c3b0ae726586ebbe16/

for the `loaded` event. If we can't remove common fields from the example events, it might make sense to shorten these `referer` URLs.